wbooks/src/app/components/AppNavigator/config.tsx

```diff
@@ -2,20 +2,31 @@ import React from 'react';
 import { Image } from 'react-native';
 import backButton from '@assets/ic_back.png';
 import headerBackground from '@assets/bc_nav_bar.png';
+import SearchBar from '@components/SearchBar';
 import colors from '@constants/colors';
 
 import styles from './styles';
 
-const customHeader = {
+const sharedHeaderProperties = {
   headerStyle: {
     height: 90
   },
-  headerTintColor: colors.white,
-  headerBackImage: () => <Image style={styles.backButton} resizeMode="cover" source={backButton} />,
   headerBackground: () => <Image style={styles.headerImage} resizeMode="stretch" source={headerBackground} />,
   headerTitleStyle: {
     textTransform: 'uppercase'
   }
 };
 
-export default customHeader;
+const mainHeader = {
+  ...sharedHeaderProperties,
+  headerTintColor: colors.white,
+  headerBackImage: () => <Image style={styles.backButton} resizeMode="cover" source={backButton} />
+};
+
+export const searchHeader = {
+  ...sharedHeaderProperties,
+  headerLeft: () => null,
```

Comment:
Why?

Reply:
If I don't use that property, the back button will be displayed while it shouldn't (according to the Marvel designs).

I obtained this idea from [here](https://stackoverflow.com/a/43980393).

![image](https://user-images.githubusercontent.com/62221379/113148147-e50a0e80-9207-11eb-8c46-8c2852d40825.png)